src/test/java/edu/pdx/cs/FakeFtpServerFactory.java

```diff
@@ -94,12 +94,11 @@ public static FakeFtpServer createServer() {
         server.setFileSystem(fileSystem);
         server.addUserAccount(aang_user);
         server.addUserAccount(bumi_user);
-        server.setServerControlPort(0);

```

Comment:
Should we revert this? It's my understanding that the port wasn't the issue with the FakeFtpServer. We can go back to defaults if that's the case.